examples/stress/README.md

```diff
@@ -0,0 +1,9 @@
+# Simple Client
+
+This folder contains simple code for stress testing the library.
+
+## How To Run It
+
+1. Build client: `go build -o stress`
```

Comment:
In our opentelemetry repo where we have end to end tests + benchmarks, we should also run this job as part for the CI (to exercise the code and to get memory stats info).

@martin-majlis-s1 If you don't plan to handle it as part of this ticket, can you please create a new one? Thanks.

Reply:
I have created that task - https://sentinelone.atlassian.net/browse/DPDV-6166